src/docks/playlistdock.ui

```diff
@@ -37,7 +37,7 @@
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
-       <number>0</number>
+       <number>1</number>
```

Comment:
Please be careful when editing a UI XML file that you do not alter the default page of a `QStackedWidget`. This must remain the same unless there is an explicit reason. Here it is a mistake.

Reply:
Yes it is a mistake, I honestly don't know how that changed and I didn't notice it. 

I will revert the file to before the commit.

 May I suggest adding a `ui->stackedWidget->setCurrentIndex(0);` to the constructor (guaranteeing it will always be the default page).